src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs

```diff
@@ -235,12 +250,17 @@ internal static void ValidateSS(IByteBuffer value, string name)
 
         internal static void ValidateTM(string value, string name)
         {
-            if (!DateTime.TryParseExact(value, DataFromatTM, CultureInfo.InvariantCulture, DateTimeStyles.NoCurrentDateDefault, out _))
+            if (!TryParseTM(value, out _))
             {
                 throw new DicomElementValidationException(name, value, DicomVR.DT, DicomCoreResource.ValueIsInvalidDate);
             }
         }
 
+        internal static bool TryParseTM(string value, out DateTime dateTime)
```

Comment:
Can we not support TM? We need to support only minimal things needed. I am not sure why would anyone index on TM

Reply:
We can add if customer needs

Reply:
removed TM